### PR TITLE
Add gcloud link

### PR DESCRIPTION
**Pull request by RayRoestenburg**
Adds more content for GCP install.

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 1]
```diff
@@ -24,10 +30,20 @@ To completely configure and use the Akka Cloud Platform you need to fill in a fe
...
 
 . Click `Deploy`.
 
+After clicking `Deploy` the Google console navigates to the `Application starting` page. Wait for the application components to deploy.
+From a terminal, login to your project using the gcloud CLI, follow the quick start at https://cloud.google.com/sdk/docs/quickstart[https://cloud.google.com/sdk/docs/quickstart {tab-icon}, window="tab"] if you are not familiar with it yet.
+Navigate to https://console.cloud.google.com/kubernetes/application, click on the cluster name for the Akka Cloud Platform application, click on the `CONNECT` button, copy the gcloud command for command line access, an example of it is shown below:
+
+[source,shell script]
+----
+gcloud container clusters get-credentials my-cluster --zone us-west1-c --project my-project
```
Comment (andreaTP) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:43]:
```suggestion
gcloud container clusters get-credentials my-cluster --zone my-zone --project my-project
```

**Review by chbatey — APPROVED** [review]

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 2]
```diff
@@ -24,10 +30,20 @@ To completely configure and use the Akka Cloud Platform you need to fill in a fe
 . The akka-operator.serviceAccount.name — This is the name of the service account used by the operator to access Kubernetes resources.
 . The Reporting Service Account — This is the service account used for reporting data to Google.
 
-. Take a moment to review all the permissions the Akka Clout Platform operator will use in order to operate your applications.
+. Take a moment to review all the permissions the Akka Cloud Platform operator will use in order to operate your applications.
 
 . Click `Deploy`.
 
+After clicking `Deploy` the Google console navigates to the `Application starting` page. Wait for the application components to deploy.
+From a terminal, login to your project using the gcloud CLI, follow the quick start at https://cloud.google.com/sdk/docs/quickstart[https://cloud.google.com/sdk/docs/quickstart {tab-icon}, window="tab"] if you are not familiar with it yet.
```
Comment (andreaTP) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:38]:
```suggestion
From a terminal, login to your project using the gcloud CLI, you can find quick-start instructions at https://cloud.google.com/sdk/docs/quickstart[https://cloud.google.com/sdk/docs/quickstart {tab-icon}, window="tab"].
```

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 3]
```diff
@@ -24,10 +30,20 @@ To completely configure and use the Akka Cloud Platform you need to fill in a fe
...
 . The Reporting Service Account — This is the service account used for reporting data to Google.
 
-. Take a moment to review all the permissions the Akka Clout Platform operator will use in order to operate your applications.
+. Take a moment to review all the permissions the Akka Cloud Platform operator will use in order to operate your applications.
 
 . Click `Deploy`.
 
+After clicking `Deploy` the Google console navigates to the `Application starting` page. Wait for the application components to deploy.
+From a terminal, login to your project using the gcloud CLI, you can find quick-start instructions at https://cloud.google.com/sdk/docs/quickstart[https://cloud.google.com/sdk/docs/quickstart {tab-icon}, window="tab"].
+Navigate to https://console.cloud.google.com/kubernetes/application, click on the cluster name for the Akka Cloud Platform application, click on the `CONNECT` button, copy the gcloud command for command line access, an example of it is shown below:
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:39]: Make these few steps a list?

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 4]
```diff
@@ -24,10 +30,20 @@ To completely configure and use the Akka Cloud Platform you need to fill in a fe
...
+
+[source,shell script]
+----
+gcloud container clusters get-credentials my-cluster --zone my-zone --project my-project
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:43]: Muktiple lines:

gcloud foo \
  --param=value

Comment (ignasi35) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc:43]: Prefer real values and 'shopping-cart' names over 'my-zone' and 'my-project'.

**Review by ignasi35 — COMMENTED** [review]
A few small nitpick s

**Review by andreaTP — APPROVED** [review]

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 5]
```diff
@@ -7,14 +7,20 @@ To install Akka Cloud Platform on Google Kubernetes Engine (GKE), you must have
 
 If you are a first time GCP user, please follow these steps:
 
-. Create your Google account at https://console.cloud.google.com[https://console.cloud.google.com {tab-icon}, window="tab"].
-. Create a project.
+. Get started with Google Cloud Platform here: https://cloud.google.com/gcp/getting-started[https://cloud.google.com/gcp/getting-started {tab-icon}, window="tab"].
+. Create your Google account at https://console.cloud.google.com[https://console.cloud.google.com {tab-icon}, window="tab"]. 
+. Sign up for an active plan and setup billing. You can start with Free tier, follow instructions at https://cloud.google.com/free[https://cloud.google.com/free {tab-icon}, window="tab"]. 
+. Create a project. 
+. Link your billing account to the project at https://console.cloud.google.com/billing/projects[https://console.cloud.google.com/billing/projects {tab-icon}, window="tab"]
```
Comment (RayRoestenburg) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:14]:
```suggestion
. Link your billing account to the project at https://console.cloud.google.com/billing/projects[https://console.cloud.google.com/billing/projects {tab-icon}, window="tab"].
```

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc (hunk 3 above)
Comment (RayRoestenburg) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc:39]: yeah I had that before... ok.

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc (hunk 4 above)
Comment (RayRoestenburg) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc:43]: ok

Comment (RayRoestenburg) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc:43]: real values depend on what the user chooses?

Comment (RayRoestenburg) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc:43]: I'll change to actual values.

**Pull request merged**